standards/naming-domains.md

```diff
@@ -4,25 +4,33 @@ category: Operating services
 # Naming domains
 
 All services with real users and which look like GOV.UK should be on
-`*.service.gov.uk` or `*.service.justice.gov.uk`.
+`*.service.gov.uk` or `*.service.justice.gov.uk`. MOJ infrastructure or
+justice-wide services may be eligible for a `*.justice.gov.uk` domain.
 
-### service.gov.uk
+### `service.gov.uk`
 
 The [service standard](https://www.gov.uk/service-manual/technology/get-a-domain-name)
 already sets out when and how to get a `*.service.gov.uk` subdomain:
 public-facing services which have passed their beta assessment should
 be able to get one.
 
-### service.justice.gov.uk
+### `service.justice.gov.uk`
 
 All other services with real users and which look like GOV.UK should be
 on here, including:
 
 - public-facing services which haven’t yet passed their beta assessment
 (but may be in private beta with a restricted set of real users)
-- internal services which look like GOV.UK (although [they can’t use the
+- internal services, which may look like GOV.UK (although [they can’t use the
```

Comment:
I had a chat with @davidread recently about exactly what "looks like GOV.UK" means - is it about using the design patterns and styles (which I think to most people would look pretty similar to GOV.UK), or more specifically about the crown, GOV.UK branding, font etc? The service manual doesn't define it explicitly.

However it's defined, do you think that concept is relevant in this standard, or should we remove the references to it? I included it initially with the aim of clarifying that SaaS services which we procure for staff to use, and don't customise styles for, don't need to be on an MOJ domain, but maybe that's obvious enough already.

Reply:
Honestly, I'd be tempted to remove all the references save the `Pitfalls of non .gov.uk domains` section at the end: I think that's useful context for why this standard exists.